lang/pl.json

```
@@ -1432,5 +1433,122 @@
   "TOKEN.MOVEMENT.ACTIONS.climb.label": "Wspinaczka",
   "TOKEN.MOVEMENT.ACTIONS.jump.label": "Skok",
   "TOKEN.MOVEMENT.ACTIONS.blink.label": "Teleport (w przestrzeni)",
-  "TOKEN.MOVEMENT.ACTIONS.displace.label": "Teleport (przez wymiary)"
+  "TOKEN.MOVEMENT.ACTIONS.displace.label": "Teleport (przez wymiary)",
+
+  "TYPES.RegionBehavior.ChaosiumCanvasInterfaceDrawingToggle": "CCI: Pokaż/ukryj rysunek",
+  "TYPES.RegionBehavior.ChaosiumCanvasInterfaceAmbientLightToggle": "CCI: Włącz/wyłącz światło",
+  "TYPES.RegionBehavior.ChaosiumCanvasInterfaceMapPinToggle": "CCI: Pokaż/ukryj pinezkę na mapie",
+  "TYPES.RegionBehavior.ChaosiumCanvasInterfaceOpenDocument": "CCI: Otwórz dokument",
+  "TYPES.RegionBehavior.ChaosiumCanvasInterfacePlaySound": "CCI: Odtwórz dźwięk",
+  "TYPES.RegionBehavior.ChaosiumCanvasInterfaceToScene": "CCI: Przejdź do sceny",
+  "TYPES.RegionBehavior.ChaosiumCanvasInterfaceTileToggle": "CCI: Pokaż/ukryj kafelek",
+
+  "CoC7.ChaosiumCanvasInterface.Actions.Hide": "Ukryj",
+  "CoC7.ChaosiumCanvasInterface.Actions.Show": "Pokaż",
+  "CoC7.ChaosiumCanvasInterface.Actions.Toggle": "Przełącz",
+  "CoC7.ChaosiumCanvasInterface.AmbientLightToggle.Button.Title": "Klawisz myszy",
+  "CoC7.ChaosiumCanvasInterface.AmbientLightToggle.Button.Hint": "Który przycisk powinien aktywować zachowanie",
+  "CoC7.ChaosiumCanvasInterface.AmbientLightToggle.Action.Title": "Akcja",
+  "CoC7.ChaosiumCanvasInterface.AmbientLightToggle.Action.Hint": "Czy powinno włączyć/wyłączyć światło",
+  "CoC7.ChaosiumCanvasInterface.AmbientLightToggle.Action.Off": "Wyłączony",
```

```suggestion
  "CoC7.ChaosiumCanvasInterface.AmbientLightToggle.Action.Off": "Wyłącz",
```
This is the correct form for action: "Wyłącz"
This is the correct form for state: "Wyłączony"